config/__init__.py

```diff
@@ -23,3 +25,8 @@ class ProfileWarning(Warning):
 user = 'NOT_SPECIFIED'
 DEBUG = False
 debug_token = 'debug-token'
+
```

Comment:
This section isn't needed since we're going to be installing this package (the profile import)